apps/web-giddh/src/app/home/components/ratio-analysis/ratio-analysis-chart.component.html

```diff
@@ -1,24 +1,23 @@
 <div id="live">
     <div class="card">
-
         <div class="panel-heading">
             <strong>Ratio Analysis</strong>
             <div class="pull-right dashboard-filter">
                 <i class="icon-refresh mr-l1" (click)="hardRefresh()"></i>
             </div>
         </div>
 
-        <div class="card-body pos-rel ratioAnalysisPanelBody">
-            <div class="dashboard-reconnectdiv" *ngIf="requestInFlight">
-                <div class="no-data mr-t2 loader-main" ng-if="errorMessage == ''">
+        <div class="card-body position-relative ratio-analysis-panel-body">
+            <div class="dashboard-reconnect-block" *ngIf="requestInFlight">
+                <div class="no-data mr-t2 loader-main" *ngIf="errorMessage == ''">
                     <!-- <div class="giddh-spinner vertical-center-spinner"></div> -->
                     <div class="spinner2">
                         <div class="cube1"></div>
                         <div class="cube2"></div>
                     </div>
                 </div>
 
-                <label class="font-black" ng-if="errorMessage != ''">{{"a"}}</label>
+                <label class="font-black" *ngIf="errorMessage != ''">{{"a"}}</label>
```

Comment:
@prateek-giddh can you please confirm that we can replace ng-if by *ngIf in this file, because in test env *ngIf is running but in prod ng-if is written.